SYCL/ESIMD/lsc/lsc_load2d_compare.cpp

```diff
@@ -0,0 +1,97 @@
+//==------------ load2d_compare.cpp - DPC++ ESIMD on-device test ----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
```

Comment:
I doubt that the test can pass with esimd_emulator.
lsc_store_2d(payload, ...) is lowered to raw_sends_store(), which is lowered to __esimd_raw_sends2_noresult() and the last one is not supported for esimd-emulator, if I did not miss something.

Please verify the test passes on emulator. If fails, please remove it from this line.